cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -48,15 +48,21 @@ func New() *cobra.Command {
 	}
 
 	tpDelCmd := &cobra.Command{
-		Use:   "delete <name>",
+		Use:   "delete <name> <namespace>",
```

Comment:
UX nit: it feels like we could use the common convention using a flag `-n <namespace>` or `--namespace <namespace>` instead of this. Like with `kubectl delete` or any `kubectl` cmd.

The slight difference is that the _implied_ namespace usually is the namespace of the current context while here it would be "global namespace" which means the cluster-wide resource.

Reply:
I don't have a strong opinion, but I'd agree with @mtardy that `--n|--namespace` is more idiomatic.